ui/app/components/clients/page/sync.hbs

```diff
@@ -2,58 +2,71 @@
-      <StatText
-        @label="Total sync clients"
-        @subText="The total number of secrets synced from Vault to other destinations during this date range."
-        @value={{this.totalUsageCounts.secret_syncs}}
-        @size="l"
-        data-test-total-sync-clients
-      />
-    </:subTitle>
-
-    <:stats>
-      <StatText
-        @label="Average sync clients per month"
-        @value={{this.average this.byMonthActivityData "secret_syncs"}}
-        @size="m"
-        class="data-details-top has-top-padding-l"
-        data-test-average-sync-clients
-      />
-    </:stats>
+      <:stats>
+        <StatText
+          @label="Average sync clients per month"
+          @value={{this.average this.byMonthActivityData "secret_syncs"}}
+          @size="m"
+          class="data-details-top has-top-padding-l"
+          data-test-average-sync-clients
+        />
+      </:stats>
 
-    <:chart>
-      <Clients::Charts::VerticalBarBasic
-        @chartTitle={{this.title}}
-        @data={{this.byMonthActivityData}}
-        @dataKey="secret_syncs"
-        @chartHeight={{200}}
-      />
-    </:chart>
+      <:chart>
+        <Clients::Charts::VerticalBarBasic
+          @chartTitle={{this.title}}
+          @data={{this.byMonthActivityData}}
+          @dataKey="secret_syncs"
+          @chartHeight={{200}}
+        />
+      </:chart>
 
-    <:emptyState>
-      <EmptyState
-        @title="No monthly secrets sync clients"
-        @subTitle="There is no secrets sync data available for this date range."
-        @bottomBorder={{true}}
-      />
-    </:emptyState>
-  </Clients::ChartContainer>
-{{else}}
-  <div class="chart-wrapper" data-test-usage-stats>
-    <div class="chart-header has-bottom-margin-m">
-      <h2 class="chart-title">{{this.title}}</h2>
-      <p class="chart-description has-bottom-padding-m">{{this.description}}</p>
+      <:emptyState>
+        <EmptyState
+          @title="No monthly secrets sync clients"
+          @subTitle="There is no secrets sync data available for this date range."
+          @bottomBorder={{true}}
+        />
+      </:emptyState>
+    </Clients::ChartContainer>
+  {{else}}
+    <div class="chart-wrapper" data-test-usage-stats>
+      <div class="chart-header has-bottom-margin-m">
+        <h2 class="chart-title">{{this.title}}</h2>
+        <p class="chart-description has-bottom-padding-m">{{this.description}}</p>
+      </div>
+      <StatText @label="Total sync clients" @value={{this.totalUsageCounts.secret_syncs}} @size="l" />
     </div>
-    <StatText @label="Total sync clients" @value={{this.totalUsageCounts.secret_syncs}} @size="l" />
-  </div>
+  {{/if}}
+{{else}}
+  <EmptyState
+    @title="No secrets sync clients"
+    @message="Secrets sync has not been activated so there is no data to be shown here."
+  >
+    <Hds::Link::Standalone
+      @icon="chevron-right"
+      @iconPosition="trailing"
+      @text="Activate secrets sync"
+      @route="sync.secrets.overview"
```

Comment:
TODO this route is broken for now; need to get the engine's route working

Reply:
I believe the route needs to have the full path. I normally search the console which prints it out on transition or the router. So here it would be `vault.cluster.sync.secrets.overview`. In the router, I found where the sync engine was mounted and got the `vault.cluster.sync` part and then within the sync engine route.js file was able to get the `secrets.overview` route.